common/common.h

```diff
@@ -309,6 +309,12 @@ struct common_params {
     enum llama_rope_scaling_type rope_scaling_type = LLAMA_ROPE_SCALING_TYPE_UNSPECIFIED;
     enum llama_pooling_type      pooling_type      = LLAMA_POOLING_TYPE_UNSPECIFIED; // pooling type for embeddings
     enum llama_attention_type    attention_type    = LLAMA_ATTENTION_TYPE_UNSPECIFIED; // attention type for embeddings
+#ifdef GGML_USE_WEBGPU
+    // FIXME the webgpu backend is lacking support for very basic operations so the test allocation for -fa auto result in an abort
+    enum llama_flash_attn_type   flash_attn_type   = LLAMA_FLASH_ATTN_TYPE_DISABLED; // whether to use Flash Attention
+#else
+    enum llama_flash_attn_type   flash_attn_type   = LLAMA_FLASH_ATTN_TYPE_AUTO; // whether to use Flash Attention
+#endif // GGML_USE_WEBGPU
```

Comment:
I think it would be better to leave the test failing than adding an exception here.